Source/SwiftLintFramework/Rules/ControlStatementRule.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ControlStatementRule.swift
+//  SwiftLint
+//
+//  Created by Andrea Mazzini on 26/05/15.
+//  Copyright (c) 2015 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct ControlStatementRule: Rule {
+    public init() {}
+
+    public let identifier = "control_statement"
+
+    public func validateFile(file: File) -> [StyleViolation] {
+        let statement = file.matchPattern("\\s{0,}(if|for|while)\\s{0,}\\(",
+            withSyntaxKinds: [.Keyword])
+        return statement.map { range in
+            return StyleViolation(type: .ControlStatement,
+                location: Location(file: file, offset: range.location),
+                severity: .Low,
+                reason: "if,for,while,do statements shouldn't wrap their conditionals in parentheses.")
```

Comment:
should probably use the regex match for the specific keyword instead of listing them all